### PR TITLE
Add Deploy step to pom

### PR DESCRIPTION
**Pull request by sreevatsanraman**
* Add steps to deploy wrangler to sonatype

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -26,6 +26,49 @@
...
+    <developer>
+      <name>Cask Data</name>
+      <email>cask-dev@googlegroups.com</email>
+      <organization>Cask Data, Inc.</organization>
+      <organizationUrl>http://www.cdap.io</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:https://github.com/caskdata/hydrator-plugins.git</connection>
```
Comment (wolf31o2) [inline pom.xml]: Fix GitHub URL

**Comment by sreevatsanraman** [comment]
Build passes locally: mvn deploy

[INFO] --- maven-deploy-plugin:2.7:deploy (default-deploy) @ wrangler ---
Downloading: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/1.0-SNAPSHOT/maven-metadata.xml
Downloaded: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/1.0-SNAPSHOT/maven-metadata.xml (768 B at 0.6 KB/sec)
Uploading: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/1.0-SNAPSHOT/wrangler-1.0-20161214.154244-2.jar
Uploaded: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/1.0-SNAPSHOT/wrangler-1.0-20161214.154244-2.jar (2064 KB at 684.5 KB/sec)
Uploading: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/1.0-SNAPSHOT/wrangler-1.0-20161214.154244-2.pom
Uploaded: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/1.0-SNAPSHOT/wrangler-1.0-20161214.154244-2.pom (13 KB at 11.3 KB/sec)
Downloading: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/maven-metadata.xml
Downloaded: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/maven-metadata.xml (282 B at 0.7 KB/sec)
Uploading: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/1.0-SNAPSHOT/maven-metadata.xml
Uploaded: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/1.0-SNAPSHOT/maven-metadata.xml (768 B at 1.0 KB/sec)
Uploading: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/maven-metadata.xml
Uploaded: https://oss.sonatype.org/content/repositories/snapshots/co/cask/hydrator/wrangler/maven-metadata.xml (282 B at 0.3 KB/sec)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16.895s
[INFO] Finished at: Wed Dec 14 07:42:51 PST 2016
[INFO] Final Memory: 68M/618M
[INFO] ------------------------------------------------------------------------

**Review by wolf31o2 — CHANGES_REQUESTED** [review]
One comment...

**Comment by sreevatsanraman** [comment]
@wolf31o2 fixed.

**Comment by wolf31o2** [comment]
:+1: LGTM

**Pull request merged**